solr/contrib/jaegertracer-configurator/src/java/org/apache/solr/JaegerTracerConfigurator.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr;
```

Comment:
Should this be in its own package? I'm not familiar with other contribs.

Reply:
Is it a requirement? Since I like a sort clean name, it can be
* org.apache.solr.JaegerTracerConfigurator
* or org.apache.solr.jaeger.TracerConfigurator

Reply:
We need a sub-package name @CaoManhDat .

Reply:
thank guys, fixed.